modules/fedimint-dummy-client/src/lib.rs

```diff
@@ -62,7 +62,7 @@ pub trait DummyClientExt {
 
 #[apply(async_trait_maybe_send!)]
 impl DummyClientExt for Client {
-    async fn print_money(&self, amount: Amount) -> anyhow::Result<()> {
+    async fn print_money(&self, amount: Amount) -> anyhow::Result<(OperationId, OutPoint)> {
```

Comment:
@elsirion we should be more consistent on when we use `OperationId` or `OutPoint` or `TransactionId`

Reply:
Typically the high-level functions exposed by the `*Ext`-traits only return `OperationId`s since internally there might be multiple transactions belonging to the same operation (e.g. LN submit offer+redeem contract). These `operation_id -> txid` mappings have to be handled internally somehow and only the operation id should be needed to query the high-level state of the operation.